iotedgehubdev/composeproject.py

```diff
@@ -4,11 +4,11 @@
 
 import json
 import os
+import io
```

Comment:
StringIO() is the only function used here, so you can change to "from io import StringIO" to stay consistent with iotedgedev